financial-templates-lib/ExpiringMultiPartyEventClient.js

```diff
@@ -0,0 +1,167 @@
+const { delay } = require("./delay");
```

Comment:
review for this file is in [PR](https://github.com/UMAprotocol/protocol/pull/1184)